ui/lib/core/addon/templates/components/replication-mode-summary.hbs

```diff
@@ -0,0 +1,101 @@
+{{#if (eq display 'menu')}}
+  <div class="level is-mobile">
+    <div class="level-left is-flex-1">
+      {{#if replicationUnsupported}}
+        Unsupported
+      {{else if replicationEnabled}}
```

Comment:
this template has a lot of conditionals in it and it can be hard to keep track of what's going on. could it be split up into smaller components?

Reply:
Yeah, before it was a couple of partials so it was easier to follow. I spent a while trying to get components to work with the larger one and couldn't get it right when using components so this (inlining the templates) was my fall back :-/ - I think pretty much this whole engine needs a rewrite. This one may be easier to divvy up than the other one I was looking at though, I'll give it a try.

Reply:
Gotcha. I think it's worth trying, but I don't see it as a blocker. Just makes things easier to read. 😬